apps/backend-sls/prisma/seed.ts

```diff
@@ -282,12 +282,62 @@ async function main(prisma: Prisma.PrismaClient) {
     update: {},
   })
 
+  const question6 = await prisma.question.upsert({
+    where: {
+      id: 5,
+    },
+    create: {
+      id: 5,
+      name: 'Numerische Testfrage',
+      content: 'Was ist richtig?',
+      contentPlain: 'Was ist richtig?',
+      type: 'NUMERICAL',
+      options: {
+        restrictions: {
+          min: 0,
+        },
+        solutionRanges: [
+          {
+            min: 0.5,
+            max: 0.6,
+          },
+          {
+            min: 2,
+          },
+        ],
+      },
+      ownerId: user.id,
+    },
+    update: {},
+  })
+
+  const question7 = await prisma.question.upsert({
+    where: {
+      id: 6,
+    },
+    create: {
+      id: 6,
+      name: 'Freitext Testfrage',
+      content: 'Was ist richtig?',
+      contentPlain: 'Was ist richtig?',
+      type: Prisma.QuestionType.FREE_TEXT,
+      options: {
+        restrictions: {
+          maxLength: 200,
+        },
+        solutions: ['Schweiz', 'CH'],
```

Comment:
Maybe: define solutions for free text as a list of strings? or no solutions at all?